src/components/widgets/sales-cycle/sales-cycle.directive.coffee

```diff
@@ -23,15 +23,23 @@ module.controller('WidgetSalesCycleCtrl', ($scope, $q, ChartFormatterSvc, $filte
   # --------------------------------------
   w.initContext = ->
     if $scope.isDataFound = angular.isDefined(w.content) && !_.isEmpty(w.content.status_average_durations)
+
+      # Takes metadata from dashboard or from widget according to 'reach' parameter
+      status_average_durations = if w.metadata.status_average_durations && w.metadata.status_average_durations.reach == 'dashboard' then ImpacDashboardsSvc.getCurrentDashboard().status_average_durations else w.metadata.status_average_durations
```

Comment:
Hi Artem,
There is still a reference to status_average_durations here / we have to look at w.metadata.status_selection